### ISSUE TITLE
Arrow through token options

### ISSUE THREAD

**Issue by ianlapham** [Uniswap/uniswap-interface#465]
Let users use arrows keys to select token

**Comment by moodysalem** [comment]
This is an accessibility thing, probably should be prioritized
